tests/rptest/tests/acls_test.py

```diff
@@ -321,6 +321,91 @@ def test_mtls_principal(self, rules=None, fail=False):
             except ClusterAuthorizationError:
                 assert fail, "list acls should have succeeded"
 
+    @cluster(num_nodes=3)
+    @parametrize(authn_method='sasl')
+    @parametrize(authn_method='mtls_identity')
+    def test_security_feature_migration(self, authn_method: str):
+        def check_perms():
+            # run a few times for good health
+            for _ in range(2):
```

Comment:
since `check_perms` is called now from `wait_until` you might consider removing the loop here and either (1) repeating the call to wait_until if the goal was to make sure that the condition continued to be true or (2) only call it once if the goal of the loop had been to catch situations in which the test hadn't yet met the target condition.

Reply:
It is bullet (2).
Done.